src/polyfill.ts

```diff
@@ -215,21 +215,21 @@ export const polyfill = (target: any, options?: PolyfillOptions) => {
 	) as Set<string>
 
 	// expand exclude options using exclusion shorthands
-	for (const excludeOption of excludeOptions) {
+	excludeOptions.forEach(function(excludeOption){
 		if (excludeOption in exclusions) {
 			for (const exclusion of exclusions[excludeOption as keyof typeof exclusions]) {
 				excludeOptions.add(exclusion)
 			}
 		}
-	}
+	});
```

Comment:
I think prettier would have caught this, but this project has an over-opinionated position of avoiding semicolons. 😉😅